build.properties

```diff
@@ -42,4 +42,4 @@ endercore_version=0.4.1.41-beta
 enderio_version=3.0.1.74_beta
 
 mantle_version = 0.10.4.+
-tico_version = 2.3.3a.jenkins271
```

Comment:
im confused too i did not delete that line all i did in that file was update forge to latest